### ISSUE TITLE
keep getting: Your project contains issues that can be fixed by running "dart fix" from the command line.

### ISSUE THREAD

**Issue by fisforfaheem** [Dart-Code/Dart-Code#5194]
Frustrated because of this notification

**Comment by DanTup** [comment]
Have you tried clicking the "Do not show again" button? This should prevent the prompt from ever showing up again. (of course, fixing the diagnostics that are triggering it for a given project will also prevent it from showing in that project unless you re-introduce issues).

If you're seeing the prompt still after clicking "Do not show again" or seeing it when you do not have anything in the Problems window that it can fix, please let me know.

**Comment by fisforfaheem** [comment]
Yes I still keep getting this even after pressing it like everytime

**Comment by DanTup** [comment]
Could you provide the info below? Thanks!

You can run the **Dart: Collect Diagnostic Information** command from the VS Code command palette (`F1`) to easily capture this information or provide it manually.

 - Operating System and version:
 - VS Code version:
 - Dart extension version:
 - Dart/Flutter SDK version:
 - Target device (if the issue relates to Flutter debugging):

**Comment by DanTup** [comment]
This came up again in https://github.com/Dart-Code/Dart-Code/issues/5202 and I was able to reproduce.

Closing this as a dupe of that (since it has some additional context), a fix is on the way.
